device-types/Server Technology/PRO3X-C3WG36RL-DQJE2NT2.yaml

```diff
@@ -0,0 +1,187 @@
+---
+manufacturer: Server Technology
+model: PRO3X-C3WG36RL-DQJE2NT2
+slug: server-technology-pro3x-c3wg36rl-dqje2nt2
+part_number: PRO3X-C3WG36RL-DQJE2NT2
+u_height: 0
+is_full_depth: false
+airflow: passive
```

Comment:
Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)

Reply:
Thanks for the feedback, 

On the datasheet, the only dimensions are volume. The only variable weight would be the power cords are fixed to device and can change overall weight. I can give a ballpark if you'd like.  

`Dimensions: 70.0in tall x 2.2in wide x 2.5in deep [1778mm x 56mm x 64mm]`

Reply:
Nope if you do not have an official weight then this can be ignored! Thank you for checking